### ISSUE TITLE
On Trial Results Page, selecting one trial to print, returning from print page, then selecting all to print displays first trial twice on print Page

### ISSUE THREAD

**Issue by sarinapadilla** [NCIOCPL/clinical-trials-search-app#571]
### Issue description
When select one trial by clicking the checkbox on the Search Results page and then click the "Print Selected" button, then the Print page loads and then if we use the browser "Go Back" option we return to the Search Results page with balls in the modal spinning and the previously selected trial checked. Now if we close the modal, we will see that the checkbox of the previously selected trial is still checked. Now if we click on the "Select all on page" checkbox and click "Print Selected" button, print page will load but we will see the trial that we first selected before selecting all trials is appearing twice on the print page.

This is definitely an edge case, but documenting here in case it's something we can investigate eventually.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  Go to https://ncigovcdode450.prod.acquia-sites.com/about-cancer/treatment/clinical-trials/search/r?lcnty=United%20States&loc=2&lst=AL&rl=2
2. Select a trial
3. Click Print Selected
4. Click the browser's back button
5. Click Select all on page
6. Click Print Selected

### What's the expected result?
- Only the 10 trials selected on the page should display for print.

### What's the actual result?
- The trial you selected first is displaying at the top, and then all of the 10 trials from selecting all on the page display after (so there is one duplicate).

### Additional details / screenshot
- ![Screenshot]()

### Related Tickets
* Issue #9999
* Issue #9999

**Comment by sarinapadilla** [comment]
Not a priority for v2 launch, but still needs to be investigated!

**Comment by blairlearn** [comment]
I'm not able to duplicate this. Looking at the print page, I only see 10 trials with no repeats. Likewise, looking at the request body, I see no duplication of trial IDs.

I do however see some very similar titles. Is it possible the "duplicate" is a misreading?

**Comment by sarinapadilla** [comment]
@blairlearn I followed the exact steps I outlined in the ticket, and was able to reproduce the issue. Here's the print page that generated:
https://www-blue-dev.cancer.gov/CTS.Print/Display?printid=6211784e-860f-ed11-a1ee-005056b388d3

Here's the trial I clicked initially, listed first:
<img width="1842" alt="Screen Shot 2022-07-29 at 5 42 48 PM" src="https://user-images.githubusercontent.com/24591940/181847037-77097abb-1a5d-4581-aeed-1e9f10f83cb3.png">

Then that trial listed again:
<img width="1848" alt="Screen Shot 2022-07-29 at 5 43 00 PM" src="https://user-images.githubusercontent.com/24591940/181847079-61f0fec0-6b48-410c-83e4-88fadf9bbdf7.png">

And then the print page showing 11 trials instead of 10:
<img width="1036" alt="Screen Shot 2022-07-29 at 5 44 04 PM" src="https://user-images.githubusercontent.com/24591940/181847106-1cc1708a-298c-4343-aba6-9c2e23aa4da2.png">

I also checked the request that's being sent to the print page, and here's the list of trial IDs given. See the first one is listed twice.
`{
    "trial_ids": [
        "NCI-2014-01507",
        "NCI-2014-01507",
        "NCI-2015-00054",
        "NCI-2019-07439",
        "NCI-2016-01041",
        "NCI-2019-01068",
        "NCI-2020-03770",
        "NCI-2020-01541",
        "NCI-2017-01672",
        "NCI-2018-02217",
        "NCI-2014-02674"
    ],
...
}`

This only happens if you select one trial, click print, click the back button, and then click "Select all on page", and then print again. Very edge case-y, but still happening for me.
